assets/stylesheets/custom.scss

```diff
@@ -93,7 +93,7 @@ h2 .value-add {
 		color: #777;
 	}
 
-	&.travis {
+	&.travis, &.testable {
```

Comment:
Line should be indented with spaces, not tabs<br>Each selector in a comma sequence should be on its own line